pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -2513,9 +2490,13 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 	})
 
 	Context("with volume", func() {
+		var vmi *v1.VirtualMachineInstance
+
+		BeforeEach(func() {
+			vmi = api.NewMinimalVMI("testvmi")
```

Comment:
Same question here for `vmi.Spec.Architecture`

Reply:
same here as well only tests required vmi.spec.architecture i added and few tests don't require thats why i didn't add in before each.